### PR TITLE
🔨 Add "Focus Breadcrumb" to command palette

### LINKED ISSUE #166213: Missing "Focus Breadcrumbs" command
Type: <b>Bug</b>

The documentation suggests using the "Focus Breadcrumbs" command (bound to `⌘.` and `Control+.` by default) here: https://code.visualstudio.com/Docs/editor/editingevolved#_breadcrumb-keyboard-navigation

However, the command is missing from the command pallette (`⌘⇧P`/`Control+Shift+P`). This makes it harder to discover and to learn the keybinding. Looking at the keyboard bindings list, the command lacks an English name, which seems to indicate it has been (accidentally?) removed from the command pallette.

This reproduces on Insiders and on the October 2022 (Version: 1.73.1).

VS Code version: Code - Insiders 1.74.0-insider (Universal) (ef415578d1b1019db585718968a303eeca4b8979, 2022-11-11T05:18:46.092Z)
OS version: Darwin arm64 21.6.0
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|6, 5, 5|
|Memory (System)|16.00GB (0.11GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30422396
pythontb:30258533
pythonptprofiler:30281269
vsdfh931cf:30280410
vshan820:30294714
pythondataviewer:30285072
vscod805cf:30301675
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30404738
cppdebug:30492333
vsclangdf:30492506
c4g48928:30535728
dsvsc012:30540252
pylantcb52:30590116
vsccc:30564266
pyindex848:30577858
nodejswelcome1:30587009
gswce1:30602633
34cci188:30599831
pyind779:30600159
dbltrim-noruby:30604474

```

</details>

### PR DESCRIPTION
**Pull request by babakks**
Fixes #166213

Just notice that I had to change the localized text ID for the `breadcrumbs.focusAndSelect` command to `cmd.focusAndSelect` (from `cmd.focus` which had to be used for the actual `breadcrumbs.focus` command).

An idea is to also include the `Focus and Select Breadcrumbs` on the command palette.

### CODE REVIEW

File: src/vs/workbench/browser/parts/editor/breadcrumbsControl.ts [hunk 1]
```diff
@@ -573,6 +573,28 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 	handler: accessor => focusAndSelectHandler(accessor, false)
 });
```
Comment (jrieken) [inline src/vs/workbench/browser/parts/editor/breadcrumbsControl.ts:574]: This call to `registerCommandAndKeybindingRule` isn't needed anymore. The new action2 also does that

**Review by jrieken — CHANGES_REQUESTED** [review]
Thanks so far. This is looking good and my feedback is about removing now obsolete code.

File: src/vs/workbench/browser/parts/editor/breadcrumbsControl.ts (hunk 1 above)
Comment (babakks) [reply src/vs/workbench/browser/parts/editor/breadcrumbsControl.ts]: I forgot to stage that part. All done.

**Review by jrieken — APPROVED** [review]
lgtm. Thanks!

**Review by joaomoreno — APPROVED** [review]

**Pull request merged**